src/ReactiveUI/Mixins/DependencyResolverMixins.cs

```diff
@@ -31,7 +31,8 @@ public static void InitializeReactiveUI(this IMutableDependencyResolver resolver
             {
                 "ReactiveUI.XamForms",
                 "ReactiveUI.Winforms",
-                "ReactiveUI.Wpf"
+                "ReactiveUI.Wpf",
+                "ReactiveUI.Uno"
```

Comment:
This will likely have to change in the near future based on talking to others.

Reply:
IMHO we should rather make a `Bootstrapper` that inherits `Application` and should be used to initialize app, and provide virtual initialization and registration stuff. Static is evil because it's sealed. That's why I am not the biggest fun of `Locator.Current`. Event it's mobile, we should improve DI performance, not use static instead. Just my opinion.
In Prism, there is an abstract service-locator, that Unity, DryIoc etc. are all ported to, and the bootstrapper can in essence be agnostic of it. We also need to make it easier to bring in modules and external V/VM assemblies and provide them with the container for type registration.

Anyway what else do we need to get this PR going so we can start using ReactiveUI in Uno apps off NuGet?

Reply:
Can I leave this one in?

Reply:
I see this now: https://github.com/reactiveui/ReactiveUI/issues/590.